chat.js

```diff
@@ -920,25 +925,55 @@ Chat.uncacheTree = function (root) {
 	} while (uncache.length > 0);
 };
 
-Chat.loadCommands = function () {
-	if (Chat.commands) return;
+Chat.loadPlugins = function () {
+	if (Chat.commands || Chat.filter) return;
 
 	FS('package.json').readTextIfExists().then(data => {
 		if (data) Chat.package = JSON.parse(data);
 	});
 
 	let baseCommands = Chat.baseCommands = require('./chat-commands').commands;
 	let commands = Chat.commands = Object.assign({}, baseCommands);
+	let chatfilters = Chat.filters = [];
```

Comment:
This should be initialized at the top-level scope.

Reply:
done